docs/installing-boot9strap-(mset9-play-store).md

```diff
@@ -58,13 +58,13 @@ In this section, you will prepare the SD card data necessary for the MSET9 explo
 1. Tap on the photo corresponding to your console model, then at the bottom, pick your current firmware version
 1. If you see the prompt of `Hax ID1 Created`, tap OK to continue
     + If you get an error, check the [troubleshooting](troubleshooting-mset9), then try again
-    + Your 3DS will appear to lose most data / no user-installed apps on HOME Menu. This is expected. Your data will come back at a later step
+    + Your console will appear to have no data / no user-installed apps on HOME Menu. **This is expected.** Your data will come back at a later step
```

Comment:
personally still prefer lose most data, as have no data at all is not correct, nand data is never effected, and this is actually asked a lot